cuckoo/machinery/virtualbox.py

```diff
@@ -357,8 +357,8 @@ def dump_memory(self, label, path):
                 "VBoxManage failed to return its version: %s" % e
             )
 
-        # VirtualBox version 4 and 5.
-        if output.startswith("5"):
+        # VirtualBox version 4, 5 and 6
+        if output.startswith("5") or output.startwith("6"):
```

Comment:
The second should be `output.startswith(...)` not `output.startwith(...)`. Furthermore, please combine the statements into `output.startswith(("5", "6"))`.